website/projects/tests/tests_aws/test_awsapitalker.py

```diff
@@ -0,0 +1,198 @@
+
+        self.assertEquals(response["CreateAccountStatus"]["AccountName"], "Test")
+        self.assertIn(("Test", "test@example.com"), [(account["Name"], account["Email"]) for account in accounts])
+
+    def test_move_account(self):
+        moto_client = boto3.client("organizations")
+
+        org_id = self.create_organization()
+
+        account_status = self.api_talker.create_account("test@example.com", "Test")
+        account_id = account_status["CreateAccountStatus"]["AccountId"]
+
+        source_ou_info = self.api_talker.create_organizational_unit(org_id, "Source OU")
+        source_ou_id = source_ou_info["OrganizationalUnit"]["Id"]
+        dest_ou_info = self.api_talker.create_organizational_unit(org_id, "Destination OU")
+        dest_ou_id = dest_ou_info["OrganizationalUnit"]["Id"]
+
+        self.api_talker.move_account(account_id, source_ou_id, dest_ou_id)
+
+        accounts_under_source = moto_client.list_children(ParentId=source_ou_id, ChildType="ACCOUNT")["Children"]
+        accounts_under_dest = moto_client.list_children(ParentId=dest_ou_id, ChildType="ACCOUNT")["Children"]
+        self.assertNotIn(account_id, [account["Id"] for account in accounts_under_source])
+        self.assertIn(account_id, [account["Id"] for account in accounts_under_dest])
+
+    def test_list_organizational_units_for_parent(self):
+        self.create_organization()
+
+        root_id = self.api_talker.list_roots()[0]["Id"]
+
+        ou_1 = self.api_talker.create_organizational_unit(root_id, "Test OU 1")
+        ou_2 = self.api_talker.create_organizational_unit(root_id, "Test OU 2")
+
+        received_ous = self.api_talker.list_organizational_units_for_parent(root_id)
+        expected_ous = [ou_1["OrganizationalUnit"], ou_2["OrganizationalUnit"]]
+
+        self.assertCountEqual(expected_ous, received_ous)
+
+    def test_list_accounts_for_parent(self):
+        self.create_organization()
+
+        self.api_talker.create_account("test1@example.com", "Test Account 1")
+        self.api_talker.create_account("test2@example.com", "Test Account 2")
+
+        root_id = self.api_talker.list_roots()[0]["Id"]
+
+        received_accounts = self.api_talker.list_accounts_for_parent(root_id)
+        received_emails = [account["Email"] for account in received_accounts]
+
+        expected_emails = ["master@example.com", "test1@example.com", "test2@example.com"]
+
+        self.assertEqual(expected_emails, received_emails)
+
+    def test_list_tags_for_resource(self):
+        org_id = self.create_organization()
+
+        specified_tags = [{"Key": "key1", "Value": "val1"}, {"Key": "key2", "Value": "val2"}]
+
+        response = self.api_talker.create_organizational_unit(org_id, "Test OU", specified_tags)
+        ou_id = response["OrganizationalUnit"]["Id"]
+
+        received_tags = self.api_talker.list_tags_for_resource(ou_id)
+
+        self.assertEqual(specified_tags, received_tags)
+
+    def test_list_roots(self):
```

Comment:
Is this a meaningful test? Could you not test if you give a organization multiple roots that your function actually gives a list with the correct number of roots. Now you make an organization which only has one root and you assert that list has length one which is quite obvious

Reply:
I could not find a boto3 function to create more roots, so I have not changed this. Since we will not need multiple roots in our project (currently we only use list_roots to get the root id of the organization), I do not think it is a big problem that we test it with just one root.